packages/react-accordion/Spec.md

```diff
@@ -0,0 +1,369 @@
+
+If you are familiar with the disclosure pattern, an accordion will feel very similar. The key distinction is that a disclosure is a standalone component that consists of a single button-panel-group. Because of this, you cannot navigate between different disclosures with a keyboard the same way you can with an accordion.
+
+## Prior art
+
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([research proposal](https://github.com/WICG/open-ui/pull/263))
+
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
+
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
+
+There's no comparison to be done between the two libraries, since v8 hasn't implemented this component.
+
+## Issues
+
+- [#16553](https://github.com/microsoft/fluentui/issues/16553)
+- [#16452](https://github.com/microsoft/fluentui/issues/16452)
+- [#12953](https://github.com/microsoft/fluentui/issues/12953)
+- [#12238](https://github.com/microsoft/fluentui/issues/12238)
+
+## API
+
+The `Accordion` should implement a `children` based API as is the standard across all the surveyed alternatives as a part of Open UI research in [Prior Art](#prior-art). The component will leverage the use of `context` in the interaction and data flows of child components.
+
+Sample usages will be given in the following section of this document [Sample code](#sample-code)
+
+### Accordion
+
+The root level component serves context and common API between all children.
+
+| Prop name          | Type                | Default Value | Details                                                                                                                    |
+| ------------------ | ------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| multiple           | boolean             | false         | Allows multiple panels to be expanded                                                                                      |
+| collapsible        | boolean             | false         | Allows multiple panels to be collapsed                                                                                     |
+| expandIconPosition | "start" or "end"    | "start"       | Position of the icon to indicate expansion                                                                                 |
```

Comment:
Will we have `expandIcon` for `Accordion`, too?

Reply:
Like a Contextual value to pass down to all `AccordionHeader` components? That's a great idea, I'll add that to the spec

Reply:
> Like a Contextual value to pass down to all AccordionHeader components? 

Yup, I noticed this as we have `expandIconPosition` there.